packages/core/crates/core-hopr/src/lib.rs

```diff
@@ -18,10 +18,6 @@ pub mod wasm {
     #[allow(unused_imports)]
     use core_path::wasm::*;
 
-    // Temporarily re-export core-path
```

Comment:
Why is it removed?

Reply:
looks like a merge conflict

Reply:
duplicate as a consequence of a merge-conflict